src/index.browser.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable */
+import IAuth0Settings from './settings';
+import { ISignInWithAuth0 } from './instance';
+
+export function useAuth0(settings: IAuth0Settings): ISignInWithAuth0 {
+  return require('./instance.browser').default(settings);
```

Comment:
This `require` could just be an import right?

Reply:
Added in follow up PR https://github.com/auth0/nextjs-auth0/pull/9